### DISCUSSION TITLE
[Question] - Does improving Dbaas support would make sense ?

### DISCUSSION THREAD

**Discussion by clun** [ing-bank/cassandra-jdbc-wrapper#29] [Ideas]
## Technical Context

As of today to use the DBAAS component on this Cassandra JDBC wrapper you need to provide a Secure connect bundle ZIP. Now using a library like the `astra-sdk-devops` it is possible to download this library on the fly and simply the URL.

## Astra JDBC Wrapper 

The work has been done in the following repository 
https://github.com/DataStax-Examples/astra-jdbc-connector/tree/main

Ending with url than looks like:
```
jdbc:astra://<db_name>/<keyspace>?token=<token>
```

## Outcomes

This issue is open for multiple purposes:

- Bring awareness to the ING Team and have their acknowledgment that we explicit mentioned them everywhere and that the component is fully open source and free as the Apache License 2.

- Bring attention to [awesome-astra.github.io ](https://awesome-astra.github.io/docs/) to see all cool integrations available for Cassandra (/Astra)

- Asked the question should those repositories be merged (I do not see an need here).

**Comment by maximevw** [comment]
Thank you for the mention and this interesting work easing the usage of the JDBC driver with Astra, especially for integrations with tools such as DataGrip, DBeaver, ...

Indeed, I think there is no need for now to merge the Astra JDBC driver into this one because this one should remain as generic as possible (I mean covering as much needs as possible to use Cassandra through the JDBC API). This is not really a good thing, in my opinion, to introduce a dependency to `astra-sdk-devops` for users not connecting to Astra. 

However, it sounds good to me to mention the Astra JDBC wrapper in the documentation for users wanting to connect to an Astra database. So, I'll add this before releasing a new version.

Since no specific action is needed for now, I will convert this issue to a discussion.

**Comment by maximevw** [comment]
Hello @clun,

I'd like to restart this discussion. The [Astra JDBC connector](https://github.com/DataStax-Examples/astra-jdbc-connector/tree/main) is a very good thing for users connecting to AstraDB, but it seems it's not really maintained (an [issue](https://github.com/DataStax-Examples/astra-jdbc-connector/issues/1) is opened without response for months), and I can understand you lack time to maintain it. 

So, it doesn't benefit from the different upgrades and bugfixes done in this project (for example, several bugs have been fixed to use it with Liquibase, ...). Consequently, it could finally interesting to "merge" both projects into this one to ease the usage of JDBC with Astra, having an up-to-date JDBC driver. What do you think about it?
